Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -2052,7 +2076,7 @@ private void AddStaticFileDependencyToFileAccessManifest(FileArtifact dependency
                 // to it. Explicitly block this, since we want inputs to not be written. Observe we already know 
                 // this is not a rewrite.
                 mask: m_excludeReportAccessMask &
-                      ~FileAccessPolicy.AllowRealInputTimestamps &
+                      (NoFakeTimestamp ? ~FileAccessPolicy.Deny : ~FileAccessPolicy.AllowRealInputTimestamps) &

```

Comment:
The expression
```csharp
NoFakeTimestamp ? ~FileAccessPolicy.Deny : ~FileAccessPolicy.AllowRealInputTimestamps
```
seems to occur at least 4 times, so maybe extract it into a property.